WL4EditorWindow.cpp

```diff
@@ -563,6 +544,10 @@ void WL4EditorWindow::RoomConfigReset(DialogParams::RoomConfigParams *currentroo
     {
         currentRoom->GetLayer(0)->CreateNewLayer_type0x10(nextroomconfig->Layer0Width, nextroomconfig->Layer0Height);
     }
+    if ((currentroomconfig->Layer0MappingTypeParam > 0x10) && (nextroomconfig->Layer0MappingTypeParam < 0x10))
```

Comment:
In this whole function, can you please add a comment before each if-else block to tell what it does (do not need before "else", just before the "if")

Reply:
Done.